src/Camelotia.Presentation.Avalonia/App.xaml

```diff
@@ -3,5 +3,17 @@
              x:Class="Camelotia.Presentation.Avalonia.App">
     <Application.Styles>
         <StyleInclude Source="avares://Citrus.Avalonia/Citrus.xaml"/>
+        <Style>
+          <Style.Resources>
+            <GeometryDrawing x:Key="ChevronRight" Brush="{DynamicResource ThemeAccentBrush}" Geometry="F1M6.3901,12.2803L5.3291,11.2193 8.5491,8.0003 5.3291,4.7803 6.3901,3.7193 10.6701,8.0003z" />
+            <GeometryDrawing x:Key="ChevronDown" Brush="{DynamicResource ThemeAccentBrush}" Geometry="F1M8,12L2.331,6.897 3.669,5.411 8,9.309 12.331,5.411 13.669,6.897z" />
+            <GeometryDrawing x:Key="ArrowUp" Brush="{DynamicResource HighlightBrush}" Geometry="M 34,57L 42,57L 42,32.25L 52,42.25L 52,31.75L 38,17.75L 24,31.75L 24,42.25L 34,32.25L 34,57 Z "/>
+            <GeometryDrawing x:Key="ArrowDown" Brush="{DynamicResource HighlightBrush}" Geometry="M 19.0002,34L 19.0002,42L 43.7502,42L 33.7502,52L 44.2502,52L 58.2502,38L 44.2502,24L 33.7502,24L 43.7502,34L 19.0002,34 Z "/>
```

Comment:
These icons make the buttons have the same widths, and IMO that looks a bit better. Thoughts?

![image](https://user-images.githubusercontent.com/6759207/92799672-a53f1f00-f3bc-11ea-9b9b-1738b3aa55fe.png)
```suggestion
            <GeometryDrawing x:Key="ArrowUp" Brush="{DynamicResource ThemeCardBrush}" Geometry="M 256,464 C 370.9,464 464,370.9 464,256 464,141.1 370.9,48 256,48 141.1,48 48,141.1 48,256 c 0,114.9 93.1,208 208,208 z m 0,-32 C 209,432 164.8,413.7 131.6,380.4 98.3,347.2 80,303 80,256 80,209 98.3,164.8 131.6,131.6 164.8,98.3 209,80 256,80 c 47,0 91.2,18.3 124.4,51.6 33.3,33.2 51.6,77.4 51.6,124.4 0,47 -18.3,91.2 -51.6,124.4 C 347.2,413.7 303,432 256,432 Z M 147.6,301.3 c -7.5,-7.5 -7.5,-19.8 0,-27.3 l 95.7,-95.4 c 7.3,-7.3 19.1,-7.5 26.6,-0.6 l 94.3,94 c 3.8,3.8 5.7,8.7 5.7,13.7 0,4.9 -1.9,9.9 -5.6,13.6 -7.5,7.5 -19.7,7.6 -27.3,0 l -81,-79.8 -81.1,81.9 c -7.5,7.5 -19.7,7.5 -27.3,-0.1 z" />
            <GeometryDrawing x:Key="ArrowDown" Brush="{DynamicResource ThemeCardBrush}" Geometry="m 48,256 c 0,114.9 93.1,208 208,208 114.9,0 208,-93.1 208,-208 C 464,141.1 370.9,48 256,48 141.1,48 48,141.1 48,256 Z m 32,0 C 80,209 98.3,164.8 131.6,131.6 164.8,98.3 209,80 256,80 c 47,0 91.2,18.3 124.4,51.6 33.3,33.2 51.6,77.4 51.6,124.4 0,47 -18.3,91.2 -51.6,124.4 C 347.2,413.7 303,432 256,432 209,432 164.8,413.7 131.6,380.4 98.3,347.2 80,303 80,256 Z M 210.7,147.6 c 7.5,-7.5 19.8,-7.5 27.3,0 l 95.4,95.7 c 7.3,7.3 7.5,19.1 0.6,26.6 l -94,94.3 c -3.8,3.8 -8.7,5.7 -13.7,5.7 -4.9,0 -9.9,-1.9 -13.6,-5.6 -7.5,-7.5 -7.6,-19.7 0,-27.3 l 79.9,-81.1 -81.9,-81.1 c -7.6,-7.4 -7.6,-19.6 0,-27.2 z" />
```

Reply:
I like it very much 👍